src/index.js

```diff
@@ -125,12 +131,16 @@ class Dropzone extends React.Component {
     }
   }
 
-  allFilesAccepted(files) {
-    return files.every(file => accepts(file, this.props.accept));
+  fileAccepted(file) {
+    return accepts(file, this.props.accept);
   }
 
-  allFilesMatchSize(files) {
-    return files.every(file => (file.size <= this.props.maxSize && file.size >= this.props.minSize));
+  fileMatchSize(file) {
+    return file.size <= this.props.maxSize && file.size >= this.props.minSize;
+  }
+
+  allFilesAccepted(files) {
+    return files.every(this.fileAccepted);
```

Comment:
I've just noticed that https://github.com/okonet/react-dropzone/pull/250/files#diff-1fdf421c05c1140f6d71444ea2b27638R135 is using `this` but the method isn't bound to this class. Not sure if this part is covered by tests but it seems that `this.props` will be always `undefined` in this context. Or am I missing something?